php/WP_CLI/Dispatcher/CompositeCommand.php

```diff
@@ -277,7 +277,7 @@ private static function get_aliases( $subcommands ) {
 	/**
 	 * Composite commands can only be known by one name.
 	 *
-	 * @return false
+	 * @return string|false
```

Comment:
Hmm this seems confusing since all this method does is return false 🧐 I wonder is this tool getting confused and considering the subcommand version of this method which extends CompositeCommand? 

https://github.com/wp-cli/wp-cli/blob/e214df253932712ebe0669d50bb485276a9a5102/php/WP_CLI/Dispatcher/Subcommand.php?plain=1#L76-L85

Reply:
Yep exactly. I wouldn't call it confused though, because it's definitely an incorrect documentation.

If the parent method returns only false, a child class returning a string is a type mismatch.